<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Basic_Settings</class>
 <widget class="QWizard" name="Basic_Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>356</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>356</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>356</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Basic Settings Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::AeroStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HelpButtonOnRight</set>
  </property>
  <property name="subTitleFormat">
   <enum>Qt::AutoText</enum>
  </property>
  <widget class="QWizardPage" name="introPage">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>Welcome to the Basic Settings Wizard</string>
   </property>
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Terrain:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">margin-left:25px;</string>
        </property>
        <property name="text">
         <string>Default height:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">margin-left:25px;</string>
        </property>
        <property name="text">
         <string>Enable draw:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="terrainHeightBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>-32768.000000000000000</double>
        </property>
        <property name="maximum">
         <double>32768.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QCheckBox" name="waterDrawBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Water:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">margin-left:25px;</string>
        </property>
        <property name="text">
         <string>Generation:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">margin-left:25px;</string>
        </property>
        <property name="text">
         <string>Import/Export:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="openImportExport">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">margin-left:25px;</string>
        </property>
        <property name="text">
         <string>Default height:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="openGeneration">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="waterHeightBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>-32768.000000000000000</double>
        </property>
        <property name="maximum">
         <double>32768.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QCheckBox" name="waterHeightEnableBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QCheckBox" name="terrainHeightEnableBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="finishPage">
   <property name="title">
    <string>Completing the Basic Settings Wizard</string>
   </property>
   <property name="subTitle">
    <string>You have sucessfully completed the Basic Settings Wizard. When you click Finish we will make changes you selected.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
